### ISSUE TITLE
Is it possible to support Mathjax in trilium?

### ISSUE THREAD

**Issue by AsciimovLee** [TriliumNext/Trilium#2179]
I found that Ckeditor5-math supports MathJax. Is it possible to add a button to select Katex or MathJax. Because I prefer MathJax to Katex.

**Comment by zadam** [comment]
ckeditor5-math supports MathJax and Katex, but not both at the same time. There might be some ways around it but IMO it's not worth it.

**Comment by linger98** [comment]
Can I pull and modify the source code to replace katex with Mathjax?

**Comment by zadam** [comment]
Yes, you need to pull `trilium-ckeditor5` repo, do changes there, build the `ckeditor.js`, pull trilium repo and replace `ckeditor.js` and build/run that repo.

**Comment by linger98** [comment]
I replaced the KaTeX with MathJax v3.2.0 — [trilium with mathjax](https://github.com/linger98/trilium-mathjax/releases/tag/v0.47.7). It has many problems, which may need to be solved until ckeditor5-math supports MathJax v3.x. Generally speaking, it does not affect the use

**Comment by linger98** [comment]
In the process of replacing KaTeX with MathJax, I encountered two rendering problems.

* The referenced notes cannot be rendered under MathJax 3.x and 2.x.
* MathJax 3.x had been rendering errors in trilium, but it can run normally in demo of Ckditor5-Math.

What is your opinion on these two issues?

**Comment by mudream4869** [comment]
Closed?
https://github.com/zadam/trilium/wiki/Text-notes#math-support

**Comment by meichthys** [comment]
[Trilium has entered maintenance mode](https://github.com/zadam/trilium/issues/4620#issuecomment-2094495698). Future enhancements will be addressed in TrilumNext: https://github.com/TriliumNext/trilium/issues/4963

**Comment by eliandoran** [comment]
We don't plan to have Mathjax support.
